lib/matplotlib/backends/backend_wx.py

```diff
@@ -1573,10 +1575,10 @@ def save_figure(self, *args):
             if ext in ('svg', 'pdf', 'ps', 'eps', 'png') and format != ext:
                 # looks like they forgot to set the image type drop
                 # down, going with the extension.
-                warnings.warn(
+                _log.warning(
```

Comment:
improper use -> `_warn_external`?

Reply:
This was changed to ```_log.warning``` following @anntzer's suggestion.

Reply:
In practice this will almost always get called through a wx GUI event handler so "external" will point to the call to mainloop(), which is not really helpful.